src/Io/ReactiveHandler.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace FrameworkX\Io;
+
+use React\EventLoop\Loop;
+use React\Http\HttpServer;
+use React\Socket\SocketServer;
+
+/**
+ * [Internal] Powerful reactive request handler built on top of ReactPHP.
+ *
+ * This is where the magic happens: The main `App` uses this class to run
+ * ReactPHP's efficient HTTP server to handle incoming HTTP requests when
+ * executed on the command line (CLI). ReactPHP's lightweight socket server can
+ * listen for a large number of concurrent connections and process multiple
+ * incoming connections simultanously. The long-running server process will
+ * continue to run until it is interrupted by a signal.
```

Comment:
Typo

```suggestion
 * incoming connections simultaneously. The long-running server process will
 * continue to run until it is interrupted by a signal.
```

Reply:
Thank you, updated as suggested! :shipit: